src/components/Main.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+import MapContainer from '../components/map/MapContainer'
+import { Layout } from 'antd';
+import Store from '../components/Store'
```

Comment:
Let's talk about removing this. There is a solution that shouldn't need an additional dependency or component files.